docs/new_deployment.md

```diff
@@ -17,20 +17,29 @@ and rename it as `<new_instance>.yaml`.
   template](../deploy/overlays/kustomization.yaml.tpl) to the new folder and
   configure the following fields:
 
-  - nameSuffix
-  - configMapGenerator[website-configmap].literals.secretProject
-  - configMapGenerator[mixer-configmap].literals.mixerProject
-  - configMapGenerator[mixer-configmap].literals.serviceName
-  - patchesStrategicMerge.spec.replicas
+  - `nameSuffix`
+  - `configMapGenerator[website-configmap].literals.secretProject`: the hosting GCP project
+  - `configMapGenerator[mixer-configmap].literals.mixerProject`: the hosting GCP project
+  - `configMapGenerator[mixer-configmap].literals.serviceName`: set this to `website-esp.endpoints.<the_hosting_GCP_project>.cloud.goog`
+  - `patchesStrategicMerge.spec.replicas`: each node in `<new_instance>.yaml` corresponds to 9 replicas.
 
-- Add additional patch to the deployment if needed.
+- Add additional patch to the deployment if needed. See [deploy/gke/overlays/karnataka/patch.yaml](../deploy/gke/overlays/karnataka/patch.yaml) and the associated changes in [kustomization.yaml](../deploy/gke/overlays/karnataka/kustomization.yaml) for an example.
 
 - In the repo root, run the following command to inspect the Kubernetes config
 
   ```bash
   kustomize build deploy/overlays/<new_instance>
   ```
 
+## Setup Pub/Sub 
```

Comment:
Callout this only applies to private instance? Same with other additions

Reply:
done